crates/bevy_text/src/pipeline.rs

```diff
@@ -14,28 +13,21 @@ use crate::{
     TextAlignment, TextSection,
 };
 
-pub struct TextPipeline<ID> {
+#[derive(Default)]
+pub struct TextPipeline {
     brush: GlyphBrush,
-    glyph_map: HashMap<ID, TextLayoutInfo>,
     map_font_id: HashMap<HandleId, FontId>,
 }
 
-impl<ID> Default for TextPipeline<ID> {
-    fn default() -> Self {
-        TextPipeline {
-            brush: GlyphBrush::default(),
-            glyph_map: Default::default(),
-            map_font_id: Default::default(),
-        }
-    }
-}
-
+/// Render information (scaled glyphs and size) for a corresponding
+/// [`Text`](crate::Text) component. Generated via [`TextPipeline::queue_text`].
+#[derive(Component, Clone, Default, Debug)]
```

Comment:
```suggestion
/// Render information for a corresponding [`Text`](crate::Text) component.
///
///  Contains scaled glyphs and their size. Generated via [`TextPipeline::queue_text`].
#[derive(Component, Clone, Default, Debug)]
```

I'm not sure it reads better?